PhysicsTools/Heppy/python/analyzers/core/AutoHandle.py

```diff
@@ -7,12 +7,13 @@ class AutoHandle( Handle, object ):
 
     handles = {}
     
-    def __init__(self, label, type, mayFail=False, fallbackLabel=None):
+    def __init__(self, label, type, mayFail=False, fallbackLabel=None, lazy=True):
         '''Note: label can be a tuple : (module_label, collection_label, process)'''
         self.label = label
         self.fallbackLabel = fallbackLabel
         self.type = type
         self.mayFail = mayFail
+	self.lazy = lazy
```

Comment:
Look ok to me on linux, but ok, can fix. What are you using in heppy,
spaces or tabs?
Giovanni
Il 28-mag-2015 16:06, "Colin Bernet" <notifications@github.com> ha scritto:

> In PhysicsTools/Heppy/python/analyzers/core/AutoHandle.py
> <https://github.com/cbernet/cmssw/pull/37#discussion_r31236108>:
>
> >          '''Note: label can be a tuple : (module_label, collection_label, process)'''
> >          self.label = label
> >          self.fallbackLabel = fallbackLabel
> >          self.type = type
> >          self.mayFail = mayFail
> > +	self.lazy = lazy
>
> Hi, indentation problem here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cbernet/cmssw/pull/37/files#r31236108>.
>